stdlib/public/core/ContiguousArrayBuffer.swift

```diff
@@ -582,38 +570,18 @@ extension _ContiguousArrayBuffer {
 }
 
 /// This is a fast implementation of _copyToNativeArrayBuffer() for collections.
-///
-/// It avoids the extra retain, release overhead from storing the
-/// ContiguousArrayBuffer into
-/// _UnsafePartiallyInitializedContiguousArrayBuffer. Since we do not support
-/// ARC loops, the extra retain, release overhead cannot be eliminated which
-/// makes assigning ranges very slow. Once this has been implemented, this code
-/// should be changed to use _UnsafePartiallyInitializedContiguousArrayBuffer.
```

Comment:
Benchmarking indicates that using `_UnsafePartiallyInitializedContigiousArrayBuffer` does not incur a measurable performance penalty. In fact, `addFrom` somehow compiles into faster code than the simple inline loop that's in the original code.